Arc/Sage/TaskListScheduleManager.swift

```diff
@@ -113,25 +113,49 @@ public class TaskListScheduleManager {
         
         let reportIds: [RSDIdentifier] = [.availability, .testSchedule]
         var successCtr = reportIds.count
+        var completedCalled = false
```

Comment:
Minor nits: feel free to merge.

It feels like a bit more communication on what is going on with completedCalled would be good. What's the point of the variable? Why do you care if it's completed?

I can infer from the code that you are trying to track if there was an error at any point while iterating through, but wouldn't errorCompleteCalled be a better name? Also moving the = true statement up into the conditional block seems slightly more correct altho admittedly a pedantic nit.

Anyway, probably fixable with a comment and/or variable name changes, but just thought this chunk was a bit more opaque than it needed to be. Fix if you think it's appropriate.

Reply:
Definitely agree, on Android I called this hasThrownError which I'll switch iOS to be as well, and I'll add some comments explaining it.